rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -32,6 +32,7 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
 
     packerCommand << absoluteTemplateFilePath
     packerCommand.removeAll([null])
+    packerCommand.removeAll("")
```

Comment:
@duftler I just did a test of this on a running instance, this is critical for rush to work. 
Base command was never used before my pull request... https://github.com/gardleopard/rosco/blob/run_aws_chroot_with_sudo/rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy#L23